### PR TITLE
missing link

### PR DESCRIPTION
**Pull request by briwylde08**

### CODE REVIEW

File: docs/encyclopedia/fees-surge-pricing-fee-strategies.mdx [hunk 1]
```diff
@@ -28,7 +28,7 @@ When the number of operations submitted to a ledger exceeds the network capacity
 
 If multiple transactions offer the same base fee during surge pricing, the transactions are shuffled randomly and the transactions at the top make the ledger. The rest of the transactions are pushed to the next ledger or discarded if they’ve been waiting for too long. If your transaction is discarded, Horizon will return a timeout error.
 
-The goal of the transaction pricing specification, which you can read in full here, is to maximize network throughput while minimizing transaction fees.
+The goal of the transaction pricing specification is to maximize network throughput while minimizing transaction fees.
```
[the remark this answers is not in the document]
Comment (ElliotFriend) [reply docs/encyclopedia/fees-surge-pricing-fee-strategies.mdx:31]:
```suggestion
The goal of the transaction pricing specification, which you can read in full [here](https://github.com/stellar/stellar-protocol/blob/master/core/cap-0005.md), is to maximize network throughput while minimizing transaction fees.
```

I took a look at the old stellar/new-docs repo (which feels weird to say lol), and it linked to CAP 5

This was the page in question: https://github.com/stellar/new-docs/blob/377f34bf821eed500a5ed0ad8a3f1ad3b13ca0be/content/docs/glossary/fees.mdx?plain=1#L43

This was the link it pointed to: https://github.com/stellar/stellar-protocol/blob/master/core/cap-0005.md

**Review by ElliotFriend — COMMENTED** [review]
Not sure where the link went, or when it disappeared, but I discovered the old docs site linked to CAP-5. I think it's still worthwhile to keep, since we do know where it should go to?

File: docs/encyclopedia/fees-surge-pricing-fee-strategies.mdx (hunk 1 above)
[the remark this answers is not in the document]
Comment (briwylde08) [reply docs/encyclopedia/fees-surge-pricing-fee-strategies.mdx:31]: You're the best

**Review by ElliotFriend — APPROVED** [review]
Looks good to me!! 👍🏻

**Pull request merged**